package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "library-main",
+  "version": "1.0.0",
+  "description": "> Awesome Books repo",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
```

Comment:
- [ ] I also noticed that from the project's requirement, you are supposed to use the `npm run start` script to run the server. That means your `webpack serve` should be assigned to a `start` script. Make sure you also fix this. e.g.:
```
...
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack",
    "dev": "webpack --mode development",
    "watch": "webpack --watch",
    "start": "webpack serve --open"
  },
...
You can now run your server with: npm run start or npm start
```

**Notice: This will not work until you didn't fix the previous issue.**